src/python/tools/config_generator.py

```diff
@@ -143,6 +156,10 @@ def set_args():
                       nargs=5, help='Dataset statistics\n' +
                       'Enter in order of num_nodes, num_relations, num_train' +
                       ' num_valid, num_test')
+    mode.add_argument('--dataset_stats_path', '-dsp',
```

Comment:
Let's remove this argument and have this dataset stats path be entirely managed by us.